ush/forecast_postdet.sh

```diff
@@ -242,12 +242,13 @@ EOF
       MM=$(printf %02d "${month}")
       ${NLN} "${FIXgfs}/aer/merra2.aerclim.2003-2014.m${MM}.nc" "aeroclim.m${MM}.nc"
     done
-    ${NLN} "${FIXgfs}/lut/optics_BC.v1_3.dat"  "${DATA}/optics_BC.dat"
-    ${NLN} "${FIXgfs}/lut/optics_OC.v1_3.dat"  "${DATA}/optics_OC.dat"
-    ${NLN} "${FIXgfs}/lut/optics_DU.v15_3.dat" "${DATA}/optics_DU.dat"
-    ${NLN} "${FIXgfs}/lut/optics_SS.v3_3.dat"  "${DATA}/optics_SS.dat"
-    ${NLN} "${FIXgfs}/lut/optics_SU.v1_3.dat"  "${DATA}/optics_SU.dat"
   fi
+    
```

Comment:
why should we move these soft links out of if-then statement?

Reply:
In other words, this is needed regardless of `IAER` values of `1011` or `2011`.

Reply:
We need to revise line 240 to
if [[ ${IAER} -eq "1011" ]] || [[ ${IAER} -eq "2011" ]]; then
and move those soft link back into if-then statement.

Reply:
probably ou is right, aeroclim* data are not needed once iaer = 2011. please ignore my suggestions.